lib/module/doom.go

```diff
@@ -0,0 +1,55 @@
+package module
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type doomStruct struct {
+	Type string `json:"type"`
+}
+
+const doomHost = "http://localhost:8000/api/player/actions"
+
+func HelpDoom() []string {
+	s := "doom <command> - play doom!"
+	return []string{s}
+}
+
+//Sanitize input
+func Doom(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	doomCommand := "enter a command, dipstick"
```

Comment:
This variable name makes no sense with what you're storing in it here. I would just move this string literal inside the `actions.Say` call.